tachyon/zk/lookup/log_derivative_halo2/prover_impl.h

```diff
@@ -238,7 +246,11 @@ BlindedPolynomial<Poly, Evals> Prover<Poly, Evals>::CreateGrandSumPoly(
   }
   size_t num_chunks = (usable_rows + chunk_size - 1) / chunk_size;
 
-  std::vector<F> segment_sum(num_chunks, F::Zero());
+  // NOTE(chokobole): This is intentional to assign
+  // |storage.table_log_derivatives| to |segment_sum| since
+  // |storage.table_log_derivatives| is not used and |segment_sum| is
+  // initialized in this loop.
```

Comment:
```suggestion
  // NOTE(chokobole): To save memory, |segment_sum| overwrites
  // |storage.table_log_derivatives| since the current values of
  // |storage.table_log_derivatives| are not needed anymore.